### PR TITLE
Add molecule scenario for stable3

### PR DESCRIPTION
**Pull request by emkll**
Added molecule scenario `workstation` that targets grsecurity's "stable3" patches (4.14 kernels) which should be suitable for workstations and provide better support for newer hardware.

4.14 workstation config is included in `files/`

### CODE REVIEW

**Comment by msheiny** [comment]
uggggh i was afraid of complicating your branch with failing tests.. ill timebox and if i cant get CI fixed ill rebased and strip my commits out onto another branch

**Comment by msheiny** [comment]
oh shit kernel is at the build process in all three scenarios !! this is so intense

File: molecule/workstation/playbook.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+        that:
+          - grsecurity_build_download_username != ''
+          - grsecurity_build_download_password != ''
+  remote_user: vagrant
+  environment:
+    USER: vagrant
+  roles:
+    - role: ansible-role-grsecurity-build
+      grsecurity_build_patch_type: stable3
+      grsecurity_build_strategy: manual
```
Comment (msheiny) [inline molecule/workstation/playbook.yml]: do we want this to be manual build strategy? I think its fine, its easy to over-ride with `molecule converge -- -e=` pragma -  I'm just making sure this is what you intended long-term.

File: molecule/workstation/tests/test_default.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+
+def test_hosts_file(host):
+    f = host.file('/etc/hosts')
+
+    assert f.exists
+    assert f.user == 'root'
+    assert f.group == 'root'
```
Comment (msheiny) [inline molecule/workstation/tests/test_default.py]: Soooo this should be removed. Looks like its from the molecule init scaffolding.

**Comment by msheiny** [comment]
rebased on master

File: molecule/workstation/playbook.yml (hunk 1 above)
Comment (emkll) [reply molecule/workstation/playbook.yml]: We could set `grsecurity_build_custom_config: 'config-workstation-4.14'` if overriding is tedious, but the manual strategy is best human usage.

**Comment by emkll** [comment]
Thanks @msheiny your CI additions look good to me, merging.

**Pull request merged**